utils/shared.js

```diff
@@ -479,13 +479,13 @@ const SSOConfig = {
 }
 
 const decodingJWT = (token) => {
-    if(token !== null || token !== undefined){
+    if (token !== null || token !== undefined) {
```

Comment:
Should this be: `if (token !== null && token !== undefined) {`

I haven't traced the JWT token, but it looks like this conditional will always execute.

Reply:
I just beautified ( in my point of view) this line, without checking context. I believe it can be shortened to `if (token) {`. Will check further.

Reply:
i would suggest this:`if (token) {`